build.cmd

```diff
@@ -1,40 +1,40 @@
-@echo off
-cd %~dp0
-
+@ECHO off
 SETLOCAL
+
+SET REPO_FOLDER=%~dp0
```

Comment:
Now that I think about it more, probably fine like this. But FYI quoting the whole assignment doesn't add double quotes to the value and protects against some special characters.